### PR TITLE
Fix agentless slack integration

### PR DESCRIPTION
**Pull request by dmitryax**
## Description

Agentless configuration doesn't work with Slack integration. Slack webhooks not triggered and you can see error in logs.   

## Logs/Alerts example

Error logs:
```
2019/06/18 20:14:19 ossec-integratord: ERROR: Unable to run integration for slack -> /var/ossec/integrations/slack
2019/06/18 20:14:19 ossec-integratord: ERROR: While running slack -> /var/ossec/integrations/slack. Output: KeyError: 'agent'
```

## Tests

- Compilation without warnings in every supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] MAC OS X
- [x] Source installation
- [x] Package installation
- [ ] Source upgrade
- [ ] Package upgrade
- Memory tests
  - [ ] Valgrind report for affected components
  - [ ] CPU impact
  - [ ] RAM usage impact
- [ ] Retrocompatibility with older Wazuh versions
- [ ] Working on cluster environments
- [ ] Configuration on demand reports new parameters
- [ ] Review logs syntax and correct language
- [ ] QA templates contemplate the added capabilities

### CODE REVIEW

File: integrations/slack [hunk 1]
```diff
@@ -60,45 +61,62 @@ def main(args):
...
-        color = "good";
-    elif ( level >= 5 and level <= 7 ):
-        color= "warning";
-    else:
-        color= "danger";
+    if (level <= 4):
+        color = "good"
+    elif (level >= 5 and level <= 7):
+        color = "warning"
+    else:
```
Comment (JuantAldea) [inline integrations/slack]: Remove this trailling whitespace

Comment (JuantAldea) [inline integrations/slack]: Remove this trailling whitespace

**Comment by JuantAldea** [comment]
Thanks @dmitryax!

**Comment by chemamartinez** [comment]
Thanks for your contribution @dmitryax,

This PR will be merged and available in the next released version.

Best regards.

**Pull request merged**